composer.json

```diff
@@ -35,7 +35,7 @@
         "sonata-project/notification-bundle": "^3.3",
         "sonata-project/seo-bundle": "^2.4",
         "sonata-project/twig-extensions": "^0.1 || ^1.2",
-        "sonata-project/user-bundle": "^4.0",
+        "sonata-project/user-bundle": "4.x-dev as 4.4",
```

Comment:
We shouldn't rely on a dev-branch either.

Do you need a new release of user-bundle now @wbloszyk ? Or maybe we wait for this https://github.com/sonata-project/SonataUserBundle/pull/1237 to be merged too ?

Reply:
We should wait for this PR. I should finish checking in tomorrow (In Monday).